pages/index.tsx

```diff
@@ -48,15 +48,18 @@ const Home: NextPage<
           >
             Deposit
           </button>
-          <button
-            className="flex font-bold items-center justify-center text-base sm:text-2xl w-18 sm:w-36 bg-transparent text-gray-400 dark:text-gray-500 disabled:text-gray-950 border-0 dark:disabled:text-white rounded-none p-2 border-b-4 border-b-transparent border-t-4 border-t-transparent disabled:border-b-gray-950 dark:disabled:border-b-white"
-            disabled={router.query.mode === 'borrow'}
-            onClick={() =>
-              router.replace('/?mode=borrow', undefined, { shallow: true })
-            }
-          >
-            Borrow
-          </button>
+          {/*TODO: remove this condition*/}
+          {(process.env.BUILD !== 'prod' && (
```

Comment:
Please use process.env.BUILD !== 'prod' ? <button>...</button> ? <></>

Using && for conditional rendering could have unwanted side effects such as the string "false" being rendered etc and is wiser to avoid altogether.